examples/step-37/doc/intro.dox

```diff
@@ -473,3 +484,44 @@ settings (on the command line, specify
 <tt>-DCMAKE_CXX_FLAGS="-march=native"</tt>, see the deal.II README for more
 information). Similar options exist for other compilers.
 
+
+<h3>Running multigrid on large-scale parallel computers</h3>
+
+As mentioned above, all components in the matrix-free framework can easily be
+parallelized with MPI using domain decomposition. Thanks to the good support
+of large-scale parallel mesh framework provided by p4est (see step-40 for
+details) in deal.II, and the fact that cell-based loops with matrix-free
+evaluation <i>only</i> need a decomposition of the mesh into chunks of roughly
+equal size on each processor, there is relatively little to do for extending
+the program to MPI. While other tutorial programs using MPI have relied on
+either PETSc or Trilinos, this program uses deal.II's own parallel vector
+facilities.
+
+The deal.II parallel vector class, LinearAlgebra::distributed::Vector, holds
+the processor-local part of the solution as well as information on and data
+fields for the ghost DoFs, i.e. DoFs that are owned by a remote processor but
```

Comment:
@davydden The vector holds all locally active DoFs but also some of the locally relevant DoFs. The selection of DoFs stored in a distributed vector initialized through `MatrixFree` is such that one can read all degrees of freedom on all locally relevant elements (locally active) plus the degrees of freedom that contraints expand into from the locally owned cells. (Most of the locally relevant but not locally active dofs would never be accessed in matrix-vector products and result in too much data sent around. I could really see this in some experiments where the mesh is not too large...)